### ISSUE TITLE
Enclosure item fails to recognize `type` of the enclosure 

### ISSUE THREAD

**Issue by deathmood** [dylang/node-rss#51]
I am parsing images from facebook and there are some image URLs like this:

https://scontent.xx.fbcdn.net/hphotos-xfp1/v/t1.0-9/s720x720/11665637_10153092749860369_5867235404683714794_n.jpg?oh=185130d3b64dca7853682ce21a9c0893&oe=562DBCC0

The `type` returned for this link is `false`. I saw that that type recognition in your lib is done via mime.lookup. Can you add ability to force specify enclosure type?

**Comment by endquote** [comment]
I ran into this too. Lines 90 and 100 should be something like:

`type : item.enclosure.type || mime.lookup(item.enclosure.file)`

**Comment by deathmood** [comment]
Yeah. But I wish such thing was in original code, because I need to create my own fork because of 1 line of code.
